Mlem/Views/Tabs/Posts/Components/Shared/Links/User Profile Link.swift

```diff
@@ -14,7 +14,24 @@ struct UserProfileLink: View
     
     @State var account: SavedAccount
     @State var user: APIPerson
-
+    
+    // Extra context about where the link is being displayed
+    // to pick the correct flair
+    @State var postContext: APIPostView? = nil
+    @State var commentContext: APIComment? = nil
+    
+    static let developerNames = [
+        "lemmy.ml/u/lFenix",
+        "vlemmy.net/u/darknavi",
+        "lemmy.ml/u/BrooklynMan"
+    ]
+    
+    static let flairDeveloper = UserProfileLinkFlair(color: Color.purple, systemIcon: "hammer.fill")
+    static let flairMod = UserProfileLinkFlair(color: Color.green, systemIcon: "shield.fill")
+    static let flairBot = UserProfileLinkFlair(color: Color.indigo, systemIcon: "server.rack")
+    static let flairOP = UserProfileLinkFlair(color: Color(red: 249 / 255, green: 105.0 / 255, blue: 14.0 / 255), systemIcon: "person.fill")
```

Comment:
we should aim to avoid defining colours like this in the code as it can become a headache for light/dark mode down the line.

If we can't use a system defined colour, we should agree some custom colours (for both light/dark) with design and add then to our asset catalogs, that way we can reference them by name throughout the project and they will automatically adjust based on the users interface setting 🙏

Reply:
I'll add them to the assets catalog, good call!